utils/create-module.js

```diff
@@ -13,7 +13,8 @@ const addModuleAsDependencyByType = (cwd, moduleName, type) => {
   const json = JSON.parse(packageContents);
 
   // Add module, sort dependencies, then write back out
-  json.dependencies[`@workday/canvas-kit-${type}-${moduleName}`] = '^3.0.0-alpha.2';
+  // `0.0.0` is special until this package is published by the canvas team
+  json.dependencies[`@workday/canvas-kit-${type}-${moduleName}`] = `0.0.0`;
```

Comment:
Oh, hrm, I grabbed the `json.version` as you had mentioned earlier. Lmk back on my PR if this would be betteer